ci/lint.yml

```
@@ -0,0 +1,15 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.15.7
```

Use latest version of go for library
```suggestion
    tag: latest
```

---

ok cool - this might need updating in other repos as well

---

This only applies to libraries